lib/rubocop/cop/rspec/factory_bot/create_list.rb

```diff
@@ -40,20 +40,23 @@ class CreateList < Base
           MSG_N_TIMES = 'Prefer %<number>s.times.'
           RESTRICT_ON_SEND = %i[create_list].freeze
 
-          # @!method n_times_block?(node)
-          def_node_matcher :n_times_block?, <<-PATTERN
+          # @!method array_new_or_n_times_block?(node)
+          def_node_matcher :array_new_or_n_times_block?, <<-PATTERN
```

Comment:
I'd use some more generic name for this, like `correctable_block?` but I leave this up to you

Reply:
I thought of that kind of name once myself, but I don't think it is necessarily correctable only by this condition.

If we create a method called `#correctable_block?`, the implementation should look something like this:

```ruby
def correctable_block?(node)
  style == :create_list &&
    array_new_or_n_times_block?(node) &&
    node.body &&
    !arguments_include_method_call?(node.body) &&
    contains_only_factory?(node.body)
end
```

Reply:
I think I learned once that method names containing "or" or "and" should be avoided, and implemented as two methods instead. Would there be significant impact (readability, performance or other) if we chose to keep `#n_times_block?` and added an `#array_new_block?` matcher?

Reply:
Originally they were split, but since we’re so similar, I suggested having just one. We haven’t come up with a good name though

Reply:
Ah, sorry about that. I really should look at the previous conversations before I comment 😅